flink-table-store-connector/src/test/java/org/apache/flink/table/store/connector/ReadWriteTableITCase.java

```diff
@@ -211,7 +212,7 @@ public void testBatchWriteWithPartitionedRecordsWithoutPk() throws Exception {
                         streamTableEnv,
                         managedTable,
                         Collections.emptyMap(),
-                        null,
+                        "dt IN ('2022-01-01', '2022-01-02')",
```

Comment:
Can you add case to cover value number more than 20?
I am not sure if there is a `in`.